syntaxsugar.md

```diff
@@ -1,5 +1,5 @@
 # Syntactic Sugar in Lua
-Syntax Sugar, in short, is a way to write something that might be more simpler to understand, or at least in an alternate way to how the language would express it by default. Lua has quite a few instances of this, which will be demonstrated below.
+Syntax Sugar, in short, is a way to write something in either a shorter way, or in a way that might be more simpler to understand to some, or at least in an alternate way to how the language would express it by default. Lua has quite a few instances of this, which will be demonstrated below.
```

Comment:
"more simpler" is redundant, just use "simpler"

Reply:
Completely agree... just need to find out what method would be the simplest and fastest to administer the change...

Reply:
You can open another PR if you want to have your name on it, or I'll do it later!

Reply:
I just rewrote that sentence, since I thought there is not much in it and it's a lot of words.